### ISSUE TITLE
Screen Features

### ISSUE THREAD

**Issue by tmitch205** [ANYCUBIC-3D/Vyper#9]
I am unable to find the information that I am looking for within this firmware..... Does this firmware show axis location in relation to 0 while in the Move Axis screen and the Z Offset screen? Adjusting the offset without it (original firmware) is a headache as the move screen just has the option of how far you can move on each button press and doesnt let you specify a specific height or location to send the head to. I want to make sure the feature is there before I install this firmware for that reason. My other printers actually allow me to put in .2mm for Z within the move screen and it sends the nozzle to where it thinks .2mm is located. 

Thanks

**Comment by BobCraig10** [comment]
No, it isnt there,. And still no display source code so the community can add it in.

**Comment by Zombie79** [comment]
there are lcd sources avalible. problem is you need to build the firmware from scratch all the images,buttons,submenus. ive been working on a firmware but i havent had much time latley to build from scratch
